doc/source/guidesandtips/gettingstartedguideforbeginnerprogrammers.rst

```diff
@@ -196,59 +196,63 @@ Here's what mine showed at the time I made this::
     [home ~]> scan-analyze 2
     ~~~~~~~~~~ Beginning scan-analyze ~~~~~~~~~~
 
-    >n00dles
+    n00dles
+    --Root Access: YES, Required hacking skill: 1
+    --Number of open ports required to NUKE: 0
+    --RAM: 4.00GB
+    
+    ----zer0
+    ------Root Access: NO, Required hacking skill: 75
+    ------Number of open ports required to NUKE: 1
+    ------RAM: 32.00GB
+    
+    foodnstuff
     --Root Access: NO, Required hacking skill: 1
     --Number of open ports required to NUKE: 0
-    --RAM: 16
-
-    >sigma-cosmetics
+    --RAM: 16.00GB
+    
+    sigma-cosmetics
     --Root Access: NO, Required hacking skill: 5
     --Number of open ports required to NUKE: 0
-    --RAM: 16
-
-    >joesguns
+    --RAM: 16.00GB
+    
+    joesguns
     --Root Access: NO, Required hacking skill: 10
     --Number of open ports required to NUKE: 0
-    --RAM: 16
-
-    ---->max-hardware
+    --RAM: 16.00GB
+    
+    ----max-hardware
     ------Root Access: NO, Required hacking skill: 80
     ------Number of open ports required to NUKE: 1
-    ------RAM: 32
-
-    >hong-fang-tea
+    ------RAM: 32.00GB
+    
+    ----CSEC
+    ------Root Access: NO, Required hacking skill: 54
+    ------Number of open ports required to NUKE: 1
+    ------RAM: 8.00GB
+    
+    hong-fang-tea
     --Root Access: NO, Required hacking skill: 30
     --Number of open ports required to NUKE: 0
-    --RAM: 16
-
-    ---->nectar-net
+    --RAM: 16.00GB
+    
+    ----nectar-net
     ------Root Access: NO, Required hacking skill: 20
     ------Number of open ports required to NUKE: 0
-    ------RAM: 16
-
-    >harakiri-sushi
+    ------RAM: 16.00GB
+    
+    harakiri-sushi
     --Root Access: NO, Required hacking skill: 40
     --Number of open ports required to NUKE: 0
-    --RAM: 16
-
-    >iron-gym
+    --RAM: 16.00GB
+    
+    iron-gym
     --Root Access: NO, Required hacking skill: 100
     --Number of open ports required to NUKE: 1
-    --RAM: 32
-
-    ---->zer0
-    ------Root Access: NO, Required hacking skill: 75
-    ------Number of open ports required to NUKE: 1
-    ------RAM: 32
-
-    ---->CSEC
-    ------Root Access: NO, Required hacking skill: 54
-    ------Number of open ports required to NUKE: 1
-    ------RAM: 8
+    --RAM: 32.00GB
 
 Take note of the following servers:
 
-* |n00dles|
```

Comment:
The references to `n00dles` from here on down shouldn't be removed, they should be replaced with `foodnstuff`.

Reply:
This is also an option which can be considered. Good point.